src/commands/refocus-collector-reregister.js

```diff
@@ -11,51 +11,34 @@
  *
  * Executes the "reregister" command.
  */
+'use strict'; // eslint-disable-line strict
 const program = require('commander');
 const logger = require('winston');
-const request = require('superagent');
-require('superagent-proxy')(request);
+const cmdUtils = require('./utils');
+const doPost = require('../utils/httpUtils.js').doPostToRefocus;
 
 program
-  .option('-n, --collectorName <collectorName>', 'The name of the collector to be started')
-  .option('-u, --refocusUrl <refocusUrl>', 'The url of the refocus instance this collector' +
-    ' will send to')
+  .option('-n, --collectorName <collectorName>', 'The name of the ' +
+    'collector to be started')
+  .option('-u, --refocusUrl <refocusUrl>', 'The url of the refocus ' +
+    'instance this collector will send to')
   .option('-t, --accessToken <accessToken>', 'A valid refocus token')
   .option('-r, --refocusProxy <refocusProxy>', 'Proxy to Refocus')
   .parse(process.argv);
 
-const collectorName = program.collectorName || process.env.RC_COLLECTOR_NAME;
-const refocusUrl = program.refocusUrl || process.env.RC_REFOCUS_URL;
-const accessToken = program.accessToken || process.env.RC_ACCESS_TOKEN;
-
-// refocusProxy is an optional arg
-const refocusProxy = program.refocusProxy || process.env.RC_REFOCUS_PROXY;
-
-if (!collectorName) {
-  logger.error('You must specify a collector name');
-  process.exit(1);
-} else if (!refocusUrl) {
-  logger.error('You must specify the url of the refocus instance');
-  process.exit(1);
-} else if (!accessToken) {
-  logger.error('You must specify an access token');
+if (!cmdUtils.validateArgs(program)) {
   process.exit(1);
 }
 
-console.log('Reregister =>', collectorName, refocusUrl, accessToken);
-const path = `/v1/collectors/${collectorName}/reregister`;
-const url = refocusUrl + path;
+const config = cmdUtils.setupConfig(program);
+const reRegisterpath = `/v1/collectors/${config.name}/reregister`;
+const refocusUrl = config.refocus.url + reRegisterpath;
+logger.log('Reregister =>', config.name, config.refocus.url + refocusUrl);
 
 // Request to Refocus to re-register collector
-const req = request.post(url)
-  .set('Authorization', accessToken);
-
-if (refocusProxy) {
-  req.proxy(refocusProxy); // set proxy for following request
-}
-
-req.then(() => {
-  logger.info(`Collector ${collectorName} has been reregistered.`);
+doPost(reRegisterpath)
+.then(() => {
+  logger.info(`Collector ${config.name} has been reregistered.`);
```

Comment:
should the message say "reregistered and started"?

Reply:
Does reregsitering a collector also start it too? I think the user will have to send a start command for it to start.